### DISCUSSION TITLE
Stream video with webpage mode

### DISCUSSION THREAD

**Discussion by nexensys** [rocksdanister/lively#1688] [Q&A] [answered]
OK so I haven't actually tried this yet, it's just an idea right now. Say I had a constant HLS stream set up on a server somewhere with an animation. Would I be able to load this with a webpage that is just a fixed, full width, full height `video` element? Or is there a way to load it directly?

**Comment by rocksdanister** [comment, marked as answer]
For browser as long as its vp9 should work (x264 codec not supported due to license.)

You can also use yt-dlp and mpv videoplayer to do this directly.
https://github.com/rocksdanister/lively/wiki/Streaming-Wallpaper
You will need to create a sample wallpaper project with LivelyInfo.json Type set as 10 to let Lively know its stream:
```json
{
  "AppVersion": "1.3.0.0",
  "Title": "Stream1",
  "Thumbnail": null,
  "Preview": null,
  "Desc": null,
  "Author": null,
  "License": null,
  "Contact": null,
  "Type": 10,
  "FileName": "url..",
  "Arguments": null,
  "IsAbsolutePath": false
}
```

**Reply by nexensys** [reply]
SICK thank you
